packages/dashboard/docker/e2e.Dockerfile

```
@@ -1,4 +1,4 @@
-FROM docker.pkg.github.com/osrf/romi-dashboard/rmf:nightly
+FROM docker.pkg.github.com/osrf/rmf-web/e2e/rmf:nightly
```

This doesn't look right. Shouldn't it be
```suggestion
FROM docker.pkg.github.com/osrf/rmf-web/rmf:nightly
```